web/packages/core/texts/tr-TR/messages.ftl

```diff
@@ -24,6 +24,7 @@ clipboard-message-description =
 clipboard-message-copy = { " " } kopyalamak için
 clipboard-message-cut = { " " } kesmek için
 clipboard-message-paste = { " " } yapıştırmak için
+error-canvas-reload = Tuval işleyici halihazırda kullanımdayken tuval oluşturucu yeniden yüklenemiyor.
```

Comment:
All good, other than this. Again, it's the missing "with", because the subject of the reloading is implicit. Perhaps we should change the source string to "... reload _the player_ with canvas renderer ..." to make it more obvious? :thinking:

Reply:
Deleted the above translation on CrowdIn after posting a comment, tagging the translator.